### DISCUSSION TITLE
Не чего не понятно.

### DISCUSSION THREAD

**Discussion by mariagithab** [community/community#38251] [New to GitHub]
Как тут добавить файл react,что бы показать проект?я как новичек даже в элементарных вещах добавить файл тут не поняла где это искать,зачем все так усложнять в простых вещах?

**Comment by DSDmark** [comment]
Hi @mariagithab,

can you specific more about what you want?
 
if you want to deploy **react** project on **github** then here is the article:-
[**gh-page**](https://github.com/gitname/react-gh-pages)

**Reply by mariagithab** [reply]
Я устала это читать такие сложности)))не ужели нельзя было все упростить у меня есть файл,я его добавляю на гитхаб,получаю ссылку на проект,и на этом все,просто удобно и понятно,а то что вы сказали это для меня понять не реально.

**Reply by DSDmark** [reply]
Hi @mariagithab,

as I'm understanding, you want to add a file in **GitHub**.
So for that there is two-way to do that:-

with **git** comments:-
[**git comment**](https://phoenixnap.com/kb/how-to-use-git)

And with **git-web-interface** like:-
![Firefox_Screenshot_2022-11-04T07-42-10 835Z](https://user-images.githubusercontent.com/85744816/199925667-6b1dc9c8-24cb-4f34-8e8f-3963cebda9b7.png)

**Reply by mariagithab** [reply]
А как мне теперь этот файл показать другому человеку?я представляла все иначе что файл загрузиться в интерпретатор,где он откроется,и я скопирую ссылку из браузера,а тут что нужно передовать?

**Comment by DSDmark** [comment]
Hi @mariagithab,

sorry, I understand a bit because of language barrier.
So, you want to show the file to another person, like than simply share the (URL). In the case of a whole folder. If you want a particular file, then click on that file and copy the permalink, or you can use URL as wall link :- 
![Screenshot 2022-11-05 at 10-06-12 RandomWaker_index html at af017bfd1e74b4a602109081fb318f67a83915c1 · DSDmark_RandomWaker](https://user-images.githubusercontent.com/85744816/200101376-d8560970-dc17-4e6f-a7c7-1df8cf51533b.png)

**Comment by samus-aran** [comment]
Hi @mariagithab, thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
